cpp/include/cugraph_c/graph.h

```diff
@@ -91,6 +93,68 @@ cugraph_error_code_t cugraph_graph_create_sg(
+ * @param [in]  src            Device array containing the source vertex ids.
+ * @param [in]  dst            Device array containing the destination vertex ids
+ * @param [in]  weights        Device array containing the edge weights.  Note that an unweighted
+ *                             graph can be created by passing weights == NULL.
+ * @param [in]  edge_ids       Device array containing the edge ids for each edge.  Optional
+                               argument that can be NULL if edge ids are not used.
+ * @param [in]  edge_type_ids  Device array containing the edge types for each edge.  Optional
+                               argument that can be NULL if edge types are not used.
+ * @param [in]  edge_start_time_ids   Device array containing the edge start times for each
+ * edge. Optional argument that can be NULL if edge start times are not used.
+ * @param [in]  edge_end_time_ids   Device array containing the edge end times for each
+ * edge. Optional argument that can be NULL if edge end times are not used.
+ * @param [in]  store_transposed If true create the graph initially in transposed format
+ * @param [in]  renumber       If true, renumber vertices to make an efficient data structure.
+ *    If false, do not renumber.  Renumbering enables some significant optimizations within
+ *    the graph primitives library, so it is strongly encouraged.  Renumbering is required if
+ *    the vertices are not sequential integer values from 0 to num_vertices.
+ * @param [in]  drop_self_loops  If true, drop any self loops that exist in the provided edge list.
+ * @param [in]  drop_multi_edges If true, drop any multi edges that exist in the provided edge list.
+ *    Note that setting this flag will arbitrarily select one instance of a multi edge to be the
+ *    edge that survives.  If the edges have properties that should be honored (e.g. sum the
+ weights,
+ *    or take the maximum weight), the caller should remove specific edges themselves and not rely
+ *    on this flag.
+ * @param [in] symmetrize      If true, symmetrize the edgelist. The symmetrization of edges
+ * with edge_ids and/or edge_type_ids is currently not supported.
+ * @param [in]  do_expensive_check    If true, do expensive checks to validate the input data
+ *    is consistent with software assumptions.  If false bypass these checks.
+ * @param [out] graph          A pointer to the graph object
+ * @param [out] error          Pointer to an error object storing details of any error.  Will
+ *                             be populated if error code is not CUGRAPH_SUCCESS
+ *
+ * @return error code
+ */
+cugraph_error_code_t cugraph_graph_create_with_times_sg(
+  const cugraph_resource_handle_t* handle,
+  const cugraph_graph_properties_t* properties,
+  const cugraph_type_erased_device_array_view_t* vertices,
+  const cugraph_type_erased_device_array_view_t* src,
+  const cugraph_type_erased_device_array_view_t* dst,
```

Comment:
Sorry for nitpicking, but better be "srcs" & "dsts" to be consistent with the "weights", "edge_ids", and so on?